ciao-cli/volume.go

```diff
@@ -62,7 +63,8 @@ The add flags are:
 
 func (cmd *volumeAddCommand) parseArgs(args []string) []string {
 	cmd.Flag.StringVar(&cmd.name, "name", "", "Volume name")
-	cmd.Flag.StringVar(&cmd.source, "source", "", "Volume ID to clone from")
+	cmd.Flag.StringVar(&cmd.sourceType, "source_type", "image", "The type of the source to clone from")
```

Comment:
Why do we need this, would an image and a volume have the same UUID? Can't we try image first then fallback to volume?